admin_guide/allocating_node_resources.adoc

```diff
@@ -11,76 +11,69 @@
 
 toc::[]
 
-== Overview
+== Purpose for allocating node resources
 
 To provide more reliable scheduling and minimize node resource overcommitment,
-each node can reserve a portion of its resources for use by all underlying
-xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#node[node
-components] (e.g., kubelet, kube-proxy, Docker) and the remaining system
-components (e.g., *sshd*, *NetworkManager*) on the host. Once specified, the
-scheduler has more information about the resources (e.g., memory, CPU) a node
-has allocated for pods.
+you can reserve a portion of the CPU and memory resources for use by the underlying xref:../architecture/infrastructure_components/kubernetes_infrastructure.adoc#node[node components] such as kubelet, kube-proxy, and the container engine. The resources that you
+reserve are also used by the remaining system components such as *sshd*, *NetworkManager*, and so on. After you specify the resources to reserve, the scheduler has more information about the remaining memory and CPU resources that a node has available for use by pods.
```

Comment:
- Looks like a force wrap landed between "that you" and "reserve"
- Our [Quick markup reference](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) suggests using backticks rather than single asterisks for system terms. I suspect that would be the case here for `sshd`, `NetworkManager`, and so on.
- For the last sentence, you might consider active voice, such as: "Specifying the resources to reserve provides the scheduler with more information about the remaining memory and CPU resources that a node has available for use by pods."